arch-chroot.in

```diff
@@ -6,7 +6,7 @@ m4_include(common)
 
 usage() {
   cat <<EOF
-usage: ${0##*/} chroot-dir [command]
+usage: ${0##*/} chroot-dir [command] [arguments]
```

Comment:
This should also be updated in the man page.

Reply:
I added another commit, which ads the required changes to docs